{{cookiecutter.project_folder}}/tasks.py

```diff
@@ -94,40 +105,57 @@ def clean(ctx, docs=True, bytecode=True, builds=True):
       'rebuild': 'True to clean all previously built docs before starting, otherwise False.',
       'doctest': 'True to run doctests, otherwise False.',
       'check_links': 'True to check all web links in docs for validity, otherwise False.'})
-def docs(ctx, doctest=False, rebuild=True, check_links=False):
+def docs(ctx, doctest=False, rebuild=False, check_links=False):
     """Builds package's HTML documentation."""
+
     if rebuild:
         clean(ctx)
+
     with chdir(BASE_FOLDER):
         if doctest:
-            testdocs(ctx, rebuild=rebuild)
+            testdocs(ctx)
+
         opts = '-E' if rebuild else ''
         ctx.run('sphinx-build {} -b html docs dist/docs'.format(opts))
+
         if check_links:
             linkcheck(ctx, rebuild=rebuild)
 
 
 @task()
-def testdocs(ctx, rebuild=True):
+def lint(ctx):
+    """Check the consistency of coding style."""
+    log.write('Running flake8 python linter...')
+    ctx.run('flake8 src')
+
+
+@task()
+def testdocs(ctx):
     """Test the examples in the docstrings."""
     log.write('Running doctest...')
-    opts = '-E' if rebuild else ''
-    ctx.run('sphinx-build {} -b doctest docs dist/docs'.format(opts))
+    ctx.run('pytest --doctest-modules')
```

Comment:
@brgcode so should be like this? I also removed rebuild argument since it's not used

Reply:
yes